test/MD051-MD060.md

```diff
@@ -1,4 +1,4 @@
-# detailed-results-MD051-MD060
+# detailed-results-MD051-MD053
```

Comment:
It's fine to leave MD060 as the top level heading. Why not add a simple example violation here? Maybe because I said I might delete these? :) I decided not to in case there's a unique test case in one of them. But I probably won't ask new rule authors to update them, I'll just do it myself.

Reply:
```suggestion
# detailed-results-MD051-MD053
```